app/src/main/AndroidManifest.xml

```diff
@@ -296,13 +296,13 @@
                 android:value=".ExampleOverviewActivity" />
         </activity>
         <activity
-            android:name=".examples.FillExtrusionActivity"
+            android:name=".examples.terrain3D.FillExtrusionActivity"
```

Comment:
Technically fill extrusion is not 3D rendering like terrain.
However logically both are 3D. Just curious - are we aligning with some other examples here (GL-JS, iOS) or is it just Android?

Reply:
We are aligning with iOS on the category. And I feel it's a good opportunity to relocate our examples in the right package instead of in the same one.

Reply:
according to audit doc here https://docs.google.com/spreadsheets/d/1zNwuUyDUqt4TPkpS4tnloG9IBrVd4lqr6ZdFOaBFmU4/edit#gid=821362685 it's supposed to be align with gl-js.

Reply:
Sounds good to me, thanks!